.github/workflows/dockerhub.yml

```
@@ -0,0 +1,19 @@
+name: Docker Image CI
```

Can we give this a better name? Maybe `Build Docker`?
```suggestion
name: Build Docker
```

---

Updated name to Build Docker Image